### ISSUE TITLE
`--patternlet inline` generates invalid code

### ISSUE THREAD

**Issue by tadeuzagallo** [nicklockwood/SwiftFormat#115]
Formatting the following minimal repro with `swiftformat --patternlet inline` generates invalid code: (version `0.26.2`)

```swift
let y : Int? = 1
let z : (Int, Int)? = (0, 3)
if let x = y, let (_, a) = z {
  print(x, a)
}
```

Generates:
```swift
let y: Int? = 1
let z: (Int, Int)? = (0, 3)
if let x = y, (_, let a) = z {
    print(x, a)
}
```

Which therefore fails with the following error:
```swift
Welcome to Apple Swift version 3.0.2 (swiftlang-800.0.63 clang-800.0.42.1). Type :help for assistance.
y: Int? = 1
z: (Int, Int)? = (0 = 0, 1 = 3)
error: repl.swift:4:15: error: expected 'let' in conditional
if let x = y, (_, let a) = z {
              ^
              let 

error: repl.swift:4:19: error: 'let' cannot appear nested inside another 'var' or 'let' pattern
if let x = y, (_, let a) = z {
                  ^
```

**Comment by nicklockwood** [comment]
Thanks, I'll take a look.

**Comment by nicklockwood** [comment]
Fixed in 0.27.0
